WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -64,13 +67,17 @@ extension FeatureFlag: OverrideableFlag {
             return "Quick Actions"
         case .meMove:
             return "Move the Me Scene to My Site"
+        case .floatingCreateButton:
+            return "Floating Create Button"
         }
     }
 
     var canOverride: Bool {
         switch self {
         case .debugMenu:
             return false
+        case .floatingCreateButton:
+            return true
```

Comment:
This should be `false`. It's still showing in the Debug menu.

<img width="400" alt="Screen Shot 2020-02-17 at 11 17 02 AM" src="https://user-images.githubusercontent.com/1816888/74678130-51968b00-5177-11ea-9c20-221d60749ae4.png">